src/hooks.js

```diff
@@ -27,7 +27,7 @@ export const useProductsQuery = ({ filters }, config) => {
     ) {
     products (
       search: $search, first: $first, last: $last, before: $before, after: $after, code_Icontains: $code, showHistory: $showHistory,
-      name_Icontains: $name, dateFrom_Gte: $dateFrom, dateTo_Lte: $dateTo, location: $location
+      name_Icontains: $name, dateFrom_Lte: $dateFrom, dateTo_Gte: $dateTo, location: $location
```

Comment:
Why change this? Seems to work already.

Reply:
This is necessary. I'll show you how it works before and after.
Before:
![prev](https://github.com/openimis/openimis-fe-product_js/assets/109145288/4f074294-0c93-4b76-affb-fa01250bc5d7)
Current: 
![current](https://github.com/openimis/openimis-fe-product_js/assets/109145288/c5b4b5b5-269c-44ac-9fd5-fe3acc4da245)
Product dateFrom is 2017-01-01. It means that it can not be used before this date. And without this change it is possible.

Reply:
It looks like it worked the other way around.